src/link/biblio/source.rs

```diff
@@ -82,14 +82,14 @@ impl BiblioEntrySource {
         }
     }
 
-    pub fn fetch_biblio_entry(&mut self, key: &str) -> BiblioEntry {
+    pub fn fetch_biblio_entry(&mut self, key: &str) -> Option<BiblioEntry> {
```

Comment:
May be worth returning a reference here?

Reply:
In bikeshed, reference and biblio entry are different (since there're two different data formats for them), so I just keep them that way.